markdown/preface.markdown

```diff
@@ -133,38 +146,33 @@ provides a unified interface to machine learning algorithms.
 provides image analysis tools that integrate cleanly with the rest of SciPy.
 
 There are others, and we will see some of them too.
-The takeaway here is that, although we will focus on NumPy and SciPy,
+Although this book will focus on NumPy and SciPy,
 it is the many surrounding packages that make Python a powerhouse for
 scientific computing.
 
-[^learn]: You can learn more about many of the tools mentioned here at
-http://www.scipy-lectures.org/, or through field-specific guides such
-as Effective Computation in Physics (http://physics.codes/).
-
 
 ## Installing Python - Anaconda
 
-Throughout this book we’re going to assume that you have Python 3.5 (or a later version) and have all the major SciPy packages installed: NumPy, SciPy, Matplotlib, IPython, scikit-image, and others.
-We will list all of the requirements and the versions we used in the environment.yml that you can find packaged with the data for this book.
+Throughout this book we’re going to assume that you have Python 3.5 (or a later version) and have all the major SciPy packages installed.
+We will list all of the requirements and the versions we used in the **environment.yml** file packaged with the data for this book.
 The easiest way to get all of these components is to install the Anaconda Python distribution.
 You can download Anaconda here: http://continuum.io/downloads. You will also find detailed installation instructions.
 
-If you find that you need to keep track of multiple versions of Python or different sets of packages, try Conda
-(if you've just downloaded the Anaconda Python distribution then you already have it!).
+If you find that you need to keep track of multiple versions of Python or different sets of packages, try conda.
```

Comment:
The MPL instructions for installing on OSX with pip are such bullshit btw. get-pip + `pip install matplotlib` will fail with an error about sudo. Then users will use sudo and eventually fuck up their system Python. There's even a warning in the [get-pip page](http://pip.readthedocs.io/en/latest/installing/), but this is completely ignored by the MPL instructions, just linked to.

<img width="696" alt="screen shot 2016-09-20 at 9 54 32 am" src="https://cloud.githubusercontent.com/assets/492549/18652753/468a599c-7f18-11e6-8832-6a146cdf98ec.png">

Hence, conda. Messing with the system Python is a terrible idea in general.